ArjunaJTA/jta/classes/com/arjuna/ats/internal/jta/recovery/arjunacore/JTATransactionLogXAResourceOrphanFilter.java

```diff
@@ -127,6 +127,7 @@ private boolean transactionLog(Xid xid)
             catch (Exception ex)
             {
                 ex.printStackTrace();
+                return true;
```

Comment:
Good catch I think the maybe is the exception so I changed it and added to jdoc. The code is only used in one place (and that is private) but its worth the change.